stubs/PyYAML/yaml/representer.pyi

```diff
@@ -1,60 +1,68 @@
+import datetime
 import sys
-from typing import Any
+from types import BuiltinFunctionType, FunctionType, ModuleType
+from typing import Any, Callable, ClassVar, Dict, List, Mapping, NoReturn, Sequence, Set, Tuple, Type, TypeVar
 
-from yaml.error import YAMLError
+from yaml.error import YAMLError as YAMLError
+from yaml.nodes import MappingNode as MappingNode, Node as Node, ScalarNode as ScalarNode, SequenceNode as SequenceNode
+
+_T = TypeVar("_T")
+_N = TypeVar("_N", bound=Node)
+_R = TypeVar("_R", bound="BaseRepresenter")
 
 class RepresenterError(YAMLError): ...
 
 class BaseRepresenter:
-    yaml_representers: Any
-    yaml_multi_representers: Any
-    default_style: Any
-    default_flow_style: Any
+    yaml_representers: ClassVar[Dict[Type[Any], Callable[["BaseRepresenter", Type[Any]], Node]]]
+    yaml_multi_representers: ClassVar[Dict[Type[Any], Callable[["BaseRepresenter", Type[Any]], Node]]]
```

Comment:
More nits: We can use build-in generics now in many cases. (Exceptions are type aliases and `tuple[x, ...]`.) Same for other uses of `Dict`, `List`, `Set`, and `Type` below. Also, quoting is not necessary in stubs:

```suggestion
    yaml_representers: ClassVar[dict[type[Any], Callable[[BaseRepresenter, type[Any]], Node]]]
    yaml_multi_representers: ClassVar[dict[type[Any], Callable[[BaseRepresenter, type[Any]], Node]]]
```